packages/shorebird_code_push_protocol/lib/src/messages/create_patch_artifact/create_patch_artifact_response.dart

```diff
@@ -24,6 +26,12 @@ class CreatePatchArtifactResponse {
   /// Converts a [CreatePatchArtifactResponse] to a Map<String, dynamic>.
   Json toJson() => _$CreatePatchArtifactResponseToJson(this);
 
+  /// The ID of the artifact;
+  final int id;
```

Comment:
needed for backward compat

Reply:
Might want to say that in the code itself. :)